spring-cloud-sleuth-core/src/test/java/org/springframework/cloud/sleuth/instrument/web/client/integration/WebClientTests.java

```diff
@@ -627,22 +626,22 @@ public void clear() {
 
 		Span span;
 
-		@RequestMapping(value = "/notrace", method = RequestMethod.GET)
+		@GetMapping("/notrace")
 		public String notrace(
-				@RequestHeader(name = "b3", required = false) String b3Single) {
-			then(b3Single).isNotNull();
+				@RequestHeader(required = false) String b3) {
```

Comment:
It's unnecessarily complicated in this case to name the parameter and variable something different. Since Rewrite can tell that there is no use of `b3Single` in the method implementation, it strips the `name` argument from the annotation, renames the parameter, and replaces all the references in the method.